proto/rill/admin/v1/api.proto

```diff
@@ -639,6 +648,7 @@ message CreateProjectRequest {
   string prod_branch = 9;
   string github_url = 10;
   map<string, string> variables = 11;
+  repeated string tags = 13;
```

Comment:
Must not be allowed as part of `CreateProject` since only superusers should be able to add tags